webapp/backend/src/handlers/api/videos.ts

```diff
@@ -26,6 +37,11 @@ demoVideosRoutes.get("/:id", (req: Request, res: Response) => {
         // Request parameter id
         const id = req.params.id;
 
+        // Check if id is a non-empty string
```

Comment:
do we not need similar checks for id that were used in metadata section here https://github.com/nsmq-ai/nsmqai/pull/18/commits/0f8ded90f2edd984ed60b5dc84e8869893ba8788#diff-5fd61999e79dcd8ccc12ecb6b6fbfda464fb914306e10ed042e9d6e6b9bcdabcR19? 

You can extract out the checks for id into a function and use it both in the metadata param endpoint and also the single video param endpoint checking as well